### ISSUE TITLE
DateTimePickerAdapter is not selecting default datetime

### ISSUE THREAD

**Issue by rafuns** [yangyxd/flutter_picker#152]
DateTimePickerAdapter is not selecting default datetime can you please help me how datetime gets selected.. 
i used 
selecteds: [6,4,1,7,2,0] 
but its not doing anything on arraypicker it works fine but on datetimepicker its not

**Comment by yasaman93** [comment]
I have the same problem

**Comment by yangyxd** [comment]
Please set 'value' to achieve the purpose.

**Comment by yasaman93** [comment]
I've just realized from another issue that the "value" argument in DateTimePickerAdapter is the answer. But it's too confusing, because I expected "selected" to work!

**Comment by yangyxd** [comment]
> 

Yes. Because datetimepicker is special, setting the initial time by specifying value is more in line with your needs.
